docs/blog/2018-12-15-develop-and-deploy-your-own-react-monorepo-app/index.md

```diff
@@ -0,0 +1,598 @@
+---
+title: Develop & Deploy Your Own React Monorepo App in Under 2 Hours, Using Lerna, Travis & Now
+date: 2018-12-15
+author: Andrico Karoulla
+tags:
+  - apps
+  - web apps
+  - applications
+  - deployment
+  - testing
+  - CI
+  - continuous integration
+excerpt: Learn how to create a balanced Lasagna instead of a tangled mess of Spaghetti.
+---
+
+## Learn how to create a balanced Lasagna instead of a tangled mess of Spaghetti.
+
+### Foreword
+
+Please put aside time for this tutorial and make sure you’ve got a good hour or two free from distractions. If I’ve done my job right, you’ll improve your general knowledge of web architecture. You’ll also be introduced to new tools whose strengths you can leverage in the wild. If you’ve used any, or all, of these tools before, then getting up and running won’t take as long. But for those unfamiliar with Lerna, Travis or Now, you’ll likely find a few traps along the way, as I did during my journey writing this tutorial. I’ve got a troubleshooting section at the end of the article for common problems, so refer to it at any time. That said, enjoy and I hope you gain a lot from this article.
+
+### Introduction
+
+If you work alone or in a small team, you’re likely to have started working on small programs that slowly grow complex and convoluted. Shit happens and you’ll soon find that your perfectly balanced Lasagna evolves into a heaped bowlful of Spaghetti.
```

Comment:
are lasagna and spaghetti capitalized for a reason?

Reply:
yes they are, the reason is that my grammar is bad :)

Reply:
Can we change "Shit" to "Crap"? I don't believe we have curse words on the Gatsby blog.